templates/hpcpack.txt

```diff
@@ -20,6 +20,7 @@ Autoscale = $Autoscale
       cyclecloud.cluster.autoscale.provisioning_timeout = 1500
       cyclecloud.cluster.autoscale.vm_retention_days = $VMRetentionDays
       cyclecloud.node.prevent_metadata_access = false
+      hpcpack.identity = $ManagedServiceIdentity
```

Comment:
Do we need to consider the backward compatibility where this parameter is not defined in clusters deployed with previous version?

Reply:
By Ben's reply in email, we don't need to worry about it.